packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1372,21 +1372,40 @@ class SvgParser {
       }
     }
 
-    // handle rgba() colors e.g. rgba(255, 255, 255, 1.0)
-    if (colorString.toLowerCase().startsWith('rgba')) {
-      final List<String> rawColorElements = colorString
+    // handle rgba() colors e.g. rgb(255, 255, 255) and rgba(255, 255, 255, 1.0)
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value/rgb
+    if (colorString.toLowerCase().startsWith('rgba') ||
+        colorString.toLowerCase().startsWith('rgb')) {
+      final List<int> rgba = colorString
           .substring(colorString.indexOf('(') + 1, colorString.indexOf(')'))
-          .split(',')
+          .split(RegExp(r'[,/\s]'))
           .map((String rawColor) => rawColor.trim())
+          .where((e) => e.isNotEmpty)
```

Comment:
This appears to be trying to correct for overly aggressive splitting (e.g., ", " as a delimiter), and will  silently accept bad input in surprising ways.

For instance, this typo where someone using legacy syntax and forgot the green value will be accepted as an unexpected color instead of erroring out: `rgba(255, , 255, 1.0)`